ansible/roles/service/hippo_delivery/files/hippo_delivery.yml

```diff
@@ -0,0 +1,285 @@
+---
+
+Description: Hippo Authoring stack
+AWSTemplateFormatVersion: 2010-09-09
+
+Parameters:
+  HostedDomainPublic:
+    Description: Environment public domain name
+    Type: String
+
+  Image:
+    Description: AMI ID of the
+    Type: AWS::EC2::Image::Id
+
+  InstanceCount:
+    Description: Number of EC2 instances to launch
+    Type: Number
+    Default:  1
+
+  InstanceMaxSize:
+    Description: Maximum number of EC2 instances to launch in Asg
+    Type: Number
+    Default: 6
+
+  InstanceMonitoring:
+    Description: Should the detailed instance monitoring be enabled
```

Comment:
Don't we want this on?

Reply:
it's extra $$ on dev and stg. We might not even need this on production.

Reply:
Got you, fine.